lua/starfall/libs_sh/players.lua

```diff
@@ -51,7 +53,7 @@ if CLIENT then
 					end
 				end
 
-				ply:SetCycle(anim.min + anim.progress * anim.range)
+				Ent_SetCycle(ply, anim.min + anim.progress * anim.range)
```

Comment:
I'm pretty sure this hasn't been localized in the file at all, wouldn't this just call a nil value?
There's a lot of this going on with a bunch of files and functions throughout the pr, from what I can tell.

Reply:
its WIP